packages/terra-functional-testing/src/commands/utils/dispatchCustomEvent.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Utility to send a custom event containing metadata.
+ * @param {string} options.name - name of event
+ * @param {object} options.metaData - metadata pertaining to event
+ */
+const dispatchCustomEvent = (options) => {
```

Comment:
This function might benefit from splitting the params:

```js
dispatchCustomEvent(name, metadata);
```

It simplifies the following:

```js
  const event = {
    name: 'applicationBase.testOverride',
    metaData: { locale: 'ar' }
  };
  
  Terra.dispatchCustomEvent(event);
  ```
  vs
  ```js
  Terra.dispatchCustomEvent('applicationBase.testOverride',  { locale: 'ar' } );
  ```
  
That said, this isn't a necessary change. If there is a chance there will be any additional options in the future the current API makes that easier to integrate.

Reply:
do we want explicit named parameters or save a couple lines? i do like knowing what each parameter explicitly is. overall, i feel meh either way

Reply:
I'd say keep it as is if there's not a strong argument to change it. I'm a little unsure about `metadata` vs `metaData`, but that's just a nit pick. Either one is probably technically correct.

> do we want explicit named parameters or save a couple lines? i do like knowing what each parameter explicitly is.

I think there are use-cases for both scenarios, but I think we should be cautious about falling into the mindset that each function we create needs an object API instead of params.

Reply:
good points - unless people have strong opinions, i'll follow the capitalization we've established in framework/application and keep it `metaData`